api/envoy/extensions/access_loggers/open_telemetry/v3alpha/logs_service.proto

```diff
@@ -29,8 +29,8 @@ message OpenTelemetryAccessLogConfig {
   // [#comment:TODO(itamarkam): add 'filter_state_objects_to_log' to logs.]
   grpc.v3.CommonGrpcAccessLogConfig common_config = 1 [(validate.rules).message = {required: true}];
 
-  // OpenTelemetry [LogResource](https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto)
-  // fields, following [Envoy access logging formatting](https://www.envoyproxy.io/docs/envoy/latest/configuration/observability/access_log/usage).
+  // OpenTelemetry `Link LogResource <https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto>`
```

Comment:
why did you prefix the link text with `Link` ?

Reply:
I misunderstood the RST syntax. Fixed.